tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -58,6 +58,9 @@ def initialize_class_vars(cls):
             # this should always be for a test project.  a check is implemented to
             # make sure this is true
             cls.project_id = ''
+            cls.dataset_id = ''
+            cls.sandbox_id = ''
+            cls.kwargs = {}
```

Comment:
Is this in order to call `clean_dataset()` in `default_test()`? If so, it seems like kwargs should live in `CleaningRulesTestBase`. Please add a comment describing what this is.

Reply:
Agree with @mark-velez on this.  All three of these are added to the base class but are only used by the inheriting class.  Do they need to be added here?  Can they be parameterized?